tests/functional/centos/7/cluster/vagrant_variables.yml

```diff
@@ -10,7 +10,7 @@ mds_vms: 1
 rgw_vms: 1
 nfs_vms: 0
 rbd_mirror_vms: 0
-client_vms: 0
+client_vms: 1
```

Comment:
You need to create a ``[clients]`` section in the hosts file for this scenario too.

I think we should add a client node to the xenial_cluster scenario as well, with ``user_config: False`` and ``openstack_config: False``.